i18n/pt-BR/pt-BR/rehike/config.i18n

```diff
@@ -0,0 +1,115 @@
+title: "Configurações do Rehike"
+saveChanges: "Salvar configurações"
+saveChangesSuccess: "Configurações atualizadas!"
+saveChangesFailure: "Erro ao atualizar as configurações."
+
+tabs:
+  appearance: "Aparência"
+  experiments: "Experimentos"
+  advanced: "Avançado"
+
+props:
+  appearance:
+    modernLogo:
+      title: "Usar a logo atual do YouTube"
+      subtitle: >
+        Usado desde 2017. Habilitar essa opção pode quebrar alguns temas feitos 
+        antes de 2017.
+    uploadButtonType:
+      title: "Tipo do botão de \"Enviar\:"

```

Comment:
Should probably be `\"Enviar\":"`. Not exactly sure what you're doing here, but if you're trying to achieve `"Enviar":` then you should use what I put there before.